docs/advanced-entity-analytics/turn-on-risk-engine.asciidoc

```diff
@@ -0,0 +1,80 @@
+[[turn-on-risk-engine]]
+== Turn on the risk scoring engine
+
+beta[]
+
+IMPORTANT: To use Entity Risk Scoring, your role must have the appropriate privileges. For more information, refer to 
+
+// <<ers-requirements, Entity Risk Scoring prerequisites>>.
```

Comment:
I've commented out the reference to prevent the build from failing, since the Entity Risk Scoring prerequisites page isn't in `main` yet. It's being introduced in https://github.com/elastic/security-docs/pull/4162, so once that PR is merged, I'll update these lines.